### DISCUSSION TITLE
Gerar PDF no flet

### DISCUSSION THREAD

**Discussion by carlosveche** [flet-dev/flet#3506] [Q&A]
Olá qual biblioteca python posso usar para gerar PDF no flet e empacotar no flutter  para rodar no Android?

**Comment by kacper-the-programmer** [comment]
first, please use english, second, please move this conversation to Q&A category

**Comment by kacper-the-programmer** [comment]
I think this article will help you: https://realpython.com/creating-modifying-pdf/

**Comment by ingdesarr1** [comment]
An option is use HTML and jinja2 to create pdf. Maybe it's not the conventional way, but it works.

**Reply by vindevoy** [reply]
This is also my approach.  I generate HTML from my Python code.  I adapt the CSS to make sure that everything looks fine on prints also.  Then I convert to PDF.

**Comment by burhansvural** [comment]
https://pypdf.readthedocs.io/en/stable/index.html
https://pypi.org/project/PyPDF2/
https://pypdf2.readthedocs.io/en/3.0.0/index.html
https://apitemplate.io/blog/how-to-generate-pdfs-from-html-with-python-pdfkit/
https://pikepdf.readthedocs.io/en/latest/
https://pypi.org/project/pdfplumber/
